src/features/components/templates/navigation/tab-nav-without-border/index.tsx

```diff
@@ -21,21 +21,23 @@ interface NavigationTabWithoutBorderType {
 
 const NavigationTabWithoutBorder = ({ tab_data }: NavigationTabWithoutBorderType) => {
     const pathname = getLocationPathname()
-    const [swiper_loading, setSwiperLoading] = useState(true)
+    const [tabs_loading, setTabsLoading] = useState(true)
```

Comment:
Its better to use `is_tabs_loading` as its a boolean state.Just a suggestion

Reply:
thanks will do it